build_aux.rs

```diff
@@ -1,11 +1,21 @@
 // Code shared by `ykrt/build.rs` and `xtask/src/main.rs`.
 
-/// Determine what kind of tracing the user has requested at compile time, by looking at RUSTFLAGS.
-/// Ideally, we'd have the user set another environment variable, and then set RUSTFLAGS
-/// accordingly, but you cant' set arbitrary RUSTFLAGS from build.rs.
+/// Determine what kind of tracing the user has requested at compile time, by looking at the output
+/// of `rustc --print cfg`. Ideally, we'd have the user set another environment variable, and then
```

Comment:
`s/another//` since we are not talking about an environment variable any more.

FWIW I much prefer this to my string parsing. Nice one!